<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="output_window">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>241</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border: 1px solid gray;
	background-color: white;
	font-size: 20px;
	qproperty-alignment: &quot;AlignVCenter | AlignRight&quot;;
	padding: 10px;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AC_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(215, 215, 215);
	border: 1px solid gray;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, x2:0, y1:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_minus_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(215, 215, 215);
	border: 1px solid gray;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, x2:0, y1:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="percent_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(215, 215, 215);
	border: 1px solid gray;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, x2:0, y1:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seven_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
background-color: white;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
									stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eight_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
background-color: white;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
									stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nine_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
background-color: white;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
									stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="four_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
background-color: white;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
									stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="five_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
background-color: white;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
									stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="six_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
background-color: white;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
									stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divide_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(215, 151, 57);
	color: white;
	border: 1px solid gray;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, x2:0, y1:0, y2:1,
									stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiple_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(215, 151, 57);
	color: white;
	border: 1px solid gray;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, x2:0, y1:0, y2:1,
									stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(215, 151, 57);
	color: white;
	border: 1px solid gray;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, x2:0, y1:0, y2:1,
									stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="one_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
background-color: white;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
									stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="two_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
background-color: white;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
									stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="three_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
background-color: white;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
									stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(215, 151, 57);
	color: white;
	border: 1px solid gray;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, x2:0, y1:0, y2:1,
									stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zero_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>121</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
background-color: white;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
									stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dot_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(215, 215, 215);
	border: 1px solid gray;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, x2:0, y1:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equals_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(215, 151, 57);
	color: white;
	border: 1px solid gray;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0, x2:0, y1:0, y2:1,
									stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
